### DISCUSSION TITLE
Getting multiple underbrackets vertically aligned

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3084] [Q&A] [answered]
When using multiple `underbrackets`, they do not get vertically aligned (at the same paper height):

`$ integral underbracket(x, G) " " underbracket((x-2)^4, F') d x $
`

![image](https://github.com/typst/typst/assets/75945439/3ad6bed9-ff25-4332-95b7-139bb1c86e04)

Is there some way of getting them vertically aligned? Thanks!

**Comment by xkevio** [comment, marked as answer]
You can try to manually adjust it with `#v(..)` but I think it is "supposed" to look like this.
```typ
$ integral underbracket(x #v(1em),  G) space underbracket((x - 2)^4, F') dif x $
```

**Reply by emilyyyylime** [reply]
I'll mention that the cause of this difference is that the first is small enough for Typst to use a single under-bracket character, while the latter is built up from multiple characters. The result is either two fonts disagreeing through a fallback or a single font that chose to compromise that aspect in favour of something else

**Comment by psads-git** [comment]
Thanks, @xkevio: That is what I was wanting!
